src/CommonDBVisible.php

```diff
@@ -309,17 +309,17 @@ public function showVisibility()
                         Html::showMassiveActionCheckBox('Group_' . $this::getType(), $data["id"]);
                         echo "</td>";
                     }
-                    echo "<td>" . Group::getTypeName(1) . "</td>";
+                    echo "<td>" . htmlspecialchars(Group::getTypeName(1)) . "</td>";
 
                     $names   = Dropdown::getDropdownName('glpi_groups', $data['groups_id'], 1);
-                    $entname = sprintf(
+                    $entname = htmlspecialchars(sprintf(
```

Comment:
The tooltip should not be escaped as it contains legitimate HTML code.
```suggestion
                    $entname = sprintf(
```